Debug/Profiler/EcsModuleViewController.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Unity.Profiling;
+using Unity.Profiling.Editor;
+using UnityEditor;
+using UnityEngine.UIElements;
+
+namespace Secs.Profiler
+{
+#if SECS_PROFILING
+
+    public sealed class EcsModuleViewController : ProfilerModuleViewController
+    {
+        private static readonly ProfilerCounterDescriptor _gcAllocatedMemoryInFrameCounterDescriptor = new ("GC Allocated In Frame", ProfilerCategory.Memory);
+        private static readonly ProfilerCounterDescriptor _scriptCounterDescriptor = new ("Scripts", ProfilerCategory.Scripts);
+        
+        private long _gcReservedBytes;
+        private long _gcUsedBytes;
+        
+        private Label _selectedFrameIndexLabel;
+        private Label _gcReservedBytesLabel;
+        private Label _gcUsedBytesLabel;
+        
+        public EcsModuleViewController(ProfilerWindow profilerWindow) : base(profilerWindow)
+        {
+            ProfilerWindow.SelectedFrameIndexChanged -= SelectedFrameIndexChanged;
+            ProfilerWindow.SelectedFrameIndexChanged += SelectedFrameIndexChanged;
+        }
+
+
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                ProfilerWindow.SelectedFrameIndexChanged -= SelectedFrameIndexChanged;
+            }
+            
+            base.Dispose(disposing);
+        }
+
+        private void SelectedFrameIndexChanged(long selectedFrameIndex)
+        {
+            var selectedFrameIndexInt32 = Convert.ToInt32(selectedFrameIndex);
```

Comment:
We are not using var with literal types. There are multiple places in the code with that issue - I won't be leaving comments for each of them, so please go through it yourself :)